### DISCUSSION TITLE
Cannot import Nuxt 3 as middleware like Nuxt 2

### DISCUSSION THREAD

**Discussion by Nikolos56651** [nuxt/nuxt#16808] [Questions] [answered]
Hi, 
I need to load my nuxt3 application into an express app, but i don't find the right commande to make it like nuxt2 (with loadNuxt and app.use(nuxt.render)).  
I have try with loadNuxt from @nuxt/kit and with buildNuxt but i can't use app.use(nuxt) because app.use require a middleware function.

**Comment by TheAlexLichter** [comment, marked as answer]
Follow along https://github.com/nuxt/framework/discussions/1704 ☺️
